sumpy/e2p.py

```diff
@@ -56,9 +56,10 @@ def __init__(self, ctx, expansion, kernels,
         if device is None:
             device = ctx.devices[0]
 
-        from sumpy.kernel import SourceDerivativeRemover, TargetDerivativeRemover
-        sdr = SourceDerivativeRemover()
-        tdr = TargetDerivativeRemover()
+        from sumpy.kernel import (SourceTransformationRemover,
+                TargetTransformationRemover)
+        sdr = SourceTransformationRemover()
+        tdr = TargetTransformationRemover()
```

Comment:
```suggestion
        sxr = SourceTransformationRemover()
        txr = TargetTransformationRemover()
```
* Nb: avoid shadowing `str` :)
* Analogous below.